client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -1,9 +1,12 @@
+import { Button, Spinner } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import { FunctionComponent } from 'react';
+import { FunctionComponent, useCallback, useState } from 'react';
```

Comment:
Should these be coming from element instead of react?

Reply:
As a best practice, yep. Missed that because sometimes I use the autocomplete 😅

Just fixed it :)